packages/react-core/src/components/Dropdown/Dropdown.tsx

```diff
@@ -54,6 +54,8 @@ export interface DropdownProps extends MenuProps, OUIAProps {
   zIndex?: number;
   /** Additional properties to pass to the Popper */
   popperProps?: DropdownPopperProps;
+  /** @beta Keys that trigger onOpenChange, defaults to tab and escape. Escape should always be included in the array, while Tab may be omitted if the menu contains non-menu items that are focusable. */
+  onOpenChangeKeys?: string[];
```

Comment:
Moved the prop upwards. The description does recommend including Escape. I think if a user wants to prevent a key from closing I think they should have the flexibility.